src/main/java/team/chisel/ctm/client/model/AbstractCTMBakedModel.java

```diff
@@ -256,39 +256,39 @@ public boolean isGui3d() {
     }
 
     @Override
-    public boolean isBuiltInRenderer() {
-        return parent.isBuiltInRenderer();
+    public boolean isCustomRenderer() {
+        return parent.isCustomRenderer();
     }
 
     @Override
-    public @Nonnull TextureAtlasSprite getParticleTexture() {
-        return this.parent.getParticleTexture();
+    public @Nonnull TextureAtlasSprite getParticleIcon() {
+        return this.parent.getParticleIcon();
     }
 
     @Override
-    public @Nonnull ItemCameraTransforms getItemCameraTransforms() {
-        return ItemCameraTransforms.DEFAULT;
+    public @Nonnull ItemTransforms getTransforms() {
+        return ItemTransforms.NO_TRANSFORMS;
     }
 
     @Override
-    public IBakedModel handlePerspective(ItemCameraTransforms.TransformType cameraTransformType, MatrixStack ms) {
-    	parent.handlePerspective(cameraTransformType, ms);
+    public BakedModel handlePerspective(ItemTransforms.TransformType cameraTransformType, PoseStack poseStack) {
+    	parent.handlePerspective(cameraTransformType, poseStack);
         return this;
     }
     
-    protected static final RenderType[] LAYERS = RenderType.getBlockRenderTypes().toArray(new RenderType[0]);
+    protected static final RenderType[] LAYERS = RenderType.chunkBufferLayers().toArray(new RenderType[0]);
     
-    protected abstract AbstractCTMBakedModel createModel(BlockState state, @Nonnull IModelCTM model, IBakedModel parent, RenderContextList ctx, Random rand);
+    protected abstract AbstractCTMBakedModel createModel(BlockState state, @Nonnull IModelCTM model, BakedModel parent, RenderContextList ctx, Random rand);
 
 	@Override
-	public boolean func_230044_c_() {
-		return getParent().func_230044_c_();
+	public boolean usesBlockLight() {
+		return getParent().usesBlockLight();
 	}
 
     private <T> T applyToParent(Random rand, Function<AbstractCTMBakedModel, T> func) {
-        IBakedModel parent = getParent(rand);
-        if (parent instanceof AbstractCTMBakedModel) {
-            return func.apply((AbstractCTMBakedModel) parent);
+        BakedModel parent = getParent(rand);
+        if (parent instanceof AbstractCTMBakedModel ctmBakedModel) {
```

Comment:
There is significant overuse of pattern matching with no thought put into variable names.
If anything this is *less* readable than the previous implementation.

Reply:
do you have any ideas for a better name or should I just remove the pattern matching there?